clouddriver-appengine/src/main/groovy/com/netflix/spinnaker/clouddriver/appengine/gcsClient/AppengineGcsRepositoryClient.groovy

```diff
@@ -46,13 +47,29 @@ class AppengineGcsRepositoryClient implements AppengineRepositoryClient {
     }
 
     def dest = targetDirectory + File.separator + applicationDirectoryRoot
-    def fullPath = repositoryUrl.substring(gsPrefix.length()) + '/' + applicationDirectoryRoot
-    def slash = fullPath.indexOf("/");
-    def bucketName = fullPath.substring(0, slash);
-    def bucketPath = fullPath.substring(slash + 1);
+
+    def fullPath = repositoryUrl.substring(gsPrefix.length())
+    if (applicationDirectoryRoot) {
+      fullPath + '/' + applicationDirectoryRoot
```

Comment:
This line looks to be missing something.

Reply:
Yeah that will be a problem, fix incoming.